### PR TITLE
Add user router middleware

### PR DESCRIPTION
**Pull request by ovekyc**
Check out this issue #127 
There is no clear way so i just check header sent property which set true after send response.

If there is better way, please let me know

### CODE REVIEW

File: src/server/users/router.js [hunk 1]
```diff
@@ -121,7 +121,14 @@ router.get('/createdposts', requiredPermission(PERMISSION.R), controller.getCrea
  *       error: "database error"
  *     }
  */
-router.get('/:id', requiredPermission(PERMISSION.R), controller.getById);
+router.get('/:id', () => {
```
Comment (hewonjeong) [inline src/server/users/router.js:124]: What is headerSent?
if controller send something like 
`res.send({message: 'success'});`
res got this 'headerSent' field?

Comment (ovekyc) [reply src/server/users/router.js:124]: Yes. After send response, headerSent property set true.
So if send response at savedPosts or createdPosts controller, It does not call ':id' controller

Comment (hewonjeong) [reply src/server/users/router.js:124]: thx!

**Comment by festinalente91** [comment]
lgtm

**Comment by hewonjeong** [comment]
LGTM

**Pull request merged**